### PR TITLE
Adds course#watch

### PR DESCRIPTION
**Pull request by felipefreitag**
* Adds course#watch, for users who bought the course
* Adds Course policy
* Changes sections#show_courses to courses#index
* Updates Course migration

Depends on #141

### CODE REVIEW

File: app/assets/stylesheets/courses/_show.sass [hunk 1]
```diff
@@ -25,3 +25,5 @@
   .purchase
     padding-top: 1rem
     padding-bottom: 2rem
+  .lesson-list
```
Comment (danielweinmann) [inline app/assets/stylesheets/courses/_show.sass]: Shouldn't this be called `lessons-list`?

File: app/controllers/courses_controller.rb [hunk 2]
```diff
@@ -2,10 +2,24 @@
 
 class CoursesController < ApplicationController
   skip_before_action :authenticate_user!
-  skip_after_action :verify_authorized
+  # skip_after_action :verify_authorized
```
Comment (danielweinmann) [inline app/controllers/courses_controller.rb]: Let's not leave commented lines on the code.

File: app/controllers/sections_controller.rb [hunk 3]
```diff
@@ -12,10 +12,9 @@ def show
     authorize @section
     if @section.name == 'Eventos & Pesquisa'
       show_events
-    elsif @section.name == 'Cursos Pangeas'
-      show_courses
     elsif @section.name == section_life
       show_section
+    else redirect_to courses_path
```
Comment (danielweinmann) [inline app/controllers/sections_controller.rb]: Why do we need this redirect?

File: app/policies/course_policy.rb [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+    true
+  end
+
+  def show?
+    true
+  end
+
+  def watch?
+    return unless user
+    user.courses.include? record
```
Comment (danielweinmann) [inline app/policies/course_policy.rb]: A very nice way of implementing this!

File: app/controllers/sections_controller.rb (hunk 3 above)
Comment (felipefreitag) [reply app/controllers/sections_controller.rb]: At sections#index, we have a list of the 3 sections. Before, the third one was rendering `sections#show_course`, but I moved that page to `courses#index` and redirected it.

File: app/policies/course_policy.rb (hunk 4 above)
Comment (felipefreitag) [reply app/policies/course_policy.rb]: Thanks, I'm learning :)

**Pull request merged**